pkg/virtctl/imageupload/imageupload.go

```diff
@@ -549,9 +549,8 @@ func createPVCSpec(size, storageClass, accessMode string, blockVolume bool) (*v1
 		spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.PersistentVolumeAccessMode(accessMode)}
 	}
 
-	if blockVolume {
```

Comment:
Can you leave the old parameter in place, mark it in the flag definition as deprecated and log a warning here that  people use an old flag?

Reply:
done.